container_pipeline/lib/command.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+import subprocess
+
+
+def run(cmd):
+    """
+    Runs a shell command.
+
+    :param cmd: Command to run
+    :type cmd: str
+
+    :return: Command output
+    :rtype: str
+    :raises: subprocess.CalledProcessError
+    """
+    return subprocess.check_output(cmd, shell=True)
```

Comment:
I'd vote for handling here so that all the methods that use this function in any way get a consistent error message and avoid the scenario where individual workers' developers have their own implementation of handling such a scenario.